tracer/test/test-applications/integrations/Samples.AWS.DynamoDBv2/AsyncHelpers.cs

```diff
@@ -0,0 +1,257 @@
+                await UpdateItemAsync(dynamoDBClient);
+                await DeleteItemAsync(dynamoDBClient);
+                
+                await PutItemsAsync(dynamoDBClient);
+                await GetItemsAsync(dynamoDBClient);
+                await DeleteItemsAsync(dynamoDBClient);
+
+                await DeleteTableAsync(dynamoDBClient);
+
+                // Needed in order to allow DynamoDB Table to be deleted
+                Thread.Sleep(1000);
+            }
+        }
+
+        public static async Task CreateTableAsync(AmazonDynamoDBClient dynamoDBClient)
+        {
+            var schema = new List<KeySchemaElement>
+            {
+                new() { AttributeName = "id", KeyType = "HASH" },
+                new() { AttributeName = "name", KeyType = "RANGE" }
+            };
+            var definitions = new List<AttributeDefinition>
+            {
+                new() { AttributeName = "id", AttributeType = "S" },
+                new() { AttributeName = "name", AttributeType = "S" }
+            };
+            var throughput = new ProvisionedThroughput
+            {
+                ReadCapacityUnits = 20,
+                WriteCapacityUnits = 50
+            };
+            var createTableRequest = new CreateTableRequest
+            {
+                TableName = TableName,
+                KeySchema = schema,
+                ProvisionedThroughput = throughput,
+                AttributeDefinitions = definitions
+            };
+
+            var response = await dynamoDBClient.CreateTableAsync(createTableRequest);
+            Console.WriteLine($"CreateTableAsync(CreateTableRequest) HTTP status code: {response.HttpStatusCode}");
+        }
+
+        public static async Task DeleteTableAsync(AmazonDynamoDBClient dynamoDBClient)
+        {
+            var deleteStreamRequest = new DeleteTableRequest { TableName = TableName };
+
+            var response = await dynamoDBClient.DeleteTableAsync(deleteStreamRequest);
+            Console.WriteLine($"DeleteTableAsync(DeleteTableRequest) HTTP status code: {response.HttpStatusCode}");
+        }
+
+        public static async Task PutItemAsync(AmazonDynamoDBClient dynamoDBClient)
+        {
+            var item = new Dictionary<string, AttributeValue>
+            {
+                ["id"] = new() { S = Guid.NewGuid().ToString() },
+                ["name"] = new() { S = "Jordan" },
+                ["lastname"] = new() { S = "González Bustamante" },
+                ["age"] = new() { N = "24" }
+            };
+
+            var putItemRequest = new PutItemRequest
+            {
+                TableName = TableName,
+                Item = item,
+            };
+
+            var response = await dynamoDBClient.PutItemAsync(putItemRequest);
+            Console.WriteLine($"PutItemAsync(PutItemRequest) HTTP status code: {response.HttpStatusCode}");
+        }
+
+        public static async Task PutItemsAsync(AmazonDynamoDBClient dynamoDBClient)
```

Comment:
nit: small little rename?

```suggestion
        public static async Task BatchWriteItemAsync(AmazonDynamoDBClient dynamoDBClient)
```

Reply:
I added these changes, left `DeleteItemsAsync` as is, since that one is a `BatchWriteItem` too.